R/clean_plot.R

```diff
@@ -0,0 +1,83 @@
+#' Clean Arcadia Plot
+#'
+#' Generalized extension function for ggplot cleanup. Right now, cleans up axis titles and removes axis lines and whitespace depending on geom type.
+#'
+#' @return A ggplot object with capitalized axis titles
+#' @import ggplot2
+#' @import snakecase
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#' library(ggplot2)
+#' library(snakecase)
```

Comment:
I think you shouldn't need to run `library(snakecase)` if it's used by the function `clean_plot()`. Loading arcadiathemeR should take care of this I think